pages/linux/iptables.md

```diff
@@ -19,6 +19,10 @@
 
 `sudo iptables -A {{chain}} -s {{ip}} -p {{protocol}} --dport {{port}} -j {{rule}}`
 
+- Add a rule to NAT all traffic from the 192.168.0.0/24 subnet to the host's public IP:
```

Comment:
```suggestion
- Add a NAT rule to translate all traffic from the `192.168.0.0/24` subnet to the host's public IP:
```
35531403820b148772e76c2e94e4856aff5ac258 should also be applied here and using Backticks for the IP will make it a different color and make it more readable.